distribution/docs/src/main/resources/content/_running/data-migration-intro.adoc

```diff
@@ -1,10 +1,10 @@
-:title: Data Migration
-:type: dataManagement
+:title: Migrating Data
+:type: dataManagementIntro
 :status: published
 :summary: Moving data/metadata to another catalog provider.
-:order: 02
+:order: 03
 
-==== ${branding} Data Migration
+== {title}
 
 Data migration is the process of moving metadata from one catalog provider to another.
```

Comment:
I recommend replacing "metadata" with "metacards". There is a field called "metadata" inside of metacards and that might be confusing.

Reply:
fixed.